css/categories.css

```diff
@@ -0,0 +1,106 @@
+@import url('./style.css');
+
+/* basic alignment categories page */
+.container {
+    padding: 0;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+/* sidebar */
+aside {
+    flex: 1 0 25%;
+    height: calc(100vh - 5rem);
+    padding: 1.5em;
+    position: sticky;
+    top: 5rem;
+    left: 0;
+    overflow-y: auto;
+    overflow-x: hidden;
```

Comment:
you can remove overflow-x, if you have specifically used overflow-y there.

Reply:
I don't want to overflow the content to the x axis that's why I have given